app/views/draws/show.html.erb

```diff
@@ -19,40 +19,32 @@
   <%= render 'ungrouped_report' %>
 </div>
 <% if policy(@draw).intent_summary? %>
-  <div class="intent-metrics">
-    <h3>Intent Metrics</h3>
-    <table>
-      <tbody>
-        <% @intent_metrics.each do |intent_value, count| %>
```

Comment:
If we're getting rid of this we should remove the controller code that calculates the metrics and the query object as well.